src/microbe_stage/organelle_unlocks/WorldBasedUnlockCondition.cs

```diff
@@ -144,17 +146,46 @@ public override void GenerateTooltip(LocalizedStringBuilder builder, IUnlockStat
         {
             var compoundName = Compound!.InternalName;
 
-            if (Min.HasValue && Max.HasValue)
+            var percentageFormat = TranslationServer.Translate("PERCENTAGE_VALUE");
```

Comment:
Considering how this is meant to generate text that can actually support being re-localized, these formattings shouldn't be applied, but rather these need to be created as more LocalizedString instances with these keys and then the values.